holohub

```diff
@@ -0,0 +1,1022 @@
+                holohub_cli_util.install_cuda_dependencies_package(
+                    "libcudnn8-dev", f"cuda{short_cuda_version}", dry_run=args.dryrun
+                )
+
+            # Install TensorRT dependencies
+            installed_libnvinferbin = subprocess.check_output(
+                ["dpkg", "--status", "libnvinfer-bin"], text=True, stderr=subprocess.DEVNULL
+            )
+            # Extract version string using regex
+            version_match = re.search(
+                r"Version: (\d+\.\d+\.\d+)\.\d+-\d+\+cuda\d+\.\d+", installed_libnvinferbin
+            )
+            if version_match:
+                installed_libnvinferbin_version = version_match.group(1)
+            else:
+                holohub_cli_util.fatal("Could not determine libnvinfer-bin version")
+
+            holohub_cli_util.install_cuda_dependencies_package(
+                "libnvinfer-headers-dev", installed_libnvinferbin_version, dry_run=args.dryrun
+            )
+            holohub_cli_util.install_cuda_dependencies_package(
+                "libnvinfer-dev", installed_libnvinferbin_version, dry_run=args.dryrun
+            )
+            holohub_cli_util.install_cuda_dependencies_package(
+                "libnvinfer-plugin-dev", installed_libnvinferbin_version, dry_run=args.dryrun
+            )
+            holohub_cli_util.install_cuda_dependencies_package(
+                "libnvonnxparsers-dev", installed_libnvinferbin_version, dry_run=args.dryrun
+            )
+
+        # Install the autocomplete
+        holohub_cli_util.run_command(
+            [
+                "sudo",
+                "cp",
+                f"{HoloHubCLI.HOLOHUB_ROOT}/utilities/holohub_autocomplete",
+                "/etc/bash_completion.d/",
+            ],
+            dry_run=args.dryrun,
+        )
+
+        print("Setup for HoloHub is ready. Happy Holocoding!")
+
+    def handle_install(self, args: argparse.Namespace) -> None:
+        """Handle install command"""
+        raise NotImplementedError("Install command not yet implemented")
+
+    def handle_clear_cache(self, args: argparse.Namespace) -> None:
+        """Handle clear-cache command"""
+        if args.dryrun:
+            print("Would clear cache folders:")
+            for pattern in ["build", "build-*", "install"]:
+                for path in host_root_dir.glob(pattern):
+                    if path.is_dir():
+                        print(f"  Would remove: {path}")
+        else:
+            print("Clearing cache...")
+            for pattern in ["build", "build-*", "install"]:
+                for path in host_root_dir.glob(pattern):
+                    if path.is_dir():
+                        shutil.rmtree(path)
+
+    def run(self) -> None:
+        """Main entry point for the CLI"""
+        args = self.parser.parse_args()
+        if hasattr(args, "func"):
+            args.func(args)
+        else:
+            self.parser.print_help()
+            sys.exit(1)
+
+
+def main():
```

Comment:
I think class `HoloHubCLI` could also live in the `utilities/cli` folder, and `holohub` only keep the main entry point